core/src/test/scala/org/http4s/rho/bits/ResponseGeneratorSpec.scala

```diff
@@ -1,53 +1,52 @@
 package org.http4s.rho.bits
 
-import org.http4s.headers.{`Content-Type`, Location, `Transfer-Encoding`, `Content-Length`}
+import org.http4s.headers.{Location, `Content-Length`, `Content-Type`, `Transfer-Encoding`}
 import org.http4s._
 import org.specs2.mutable.Specification
-
-
+import scodec.bits.ByteVector
 
 class ResponseGeneratorSpec extends Specification {
   import ResponseGeneratorInstances._
 
   "ResponseGenerator" should {
     "Build a response with a body" in {
-      val result = Ok("Foo").run
+      val result = Ok("Foo").unsafeRun
```

Comment:
IMHO, thats out of the scope of this PR, the purpose was to get Rho updated for Http4s.  I agree we need to clean up A LOT of the tests, but I didn't want to change any of the behaviors of the code in this PR.